.editorconfig

```diff
@@ -0,0 +1,31 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
```

Comment:
Yes and the fact that each editor uses different amount space for tabs is creating some differences in PRs :(